data/json/requirements/melee.json

```diff
@@ -0,0 +1,682 @@
+        [ "ch_battleaxe", -1 ],
+        [ "qt_battleaxe", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_batons",
+    "type": "requirement",
+    "//": "Any basic, easy-to-use bashing weapons.  Like police batons, sticks, or baseball bats.",
+    "tools": [
+      [
+        [ "cudgel", -1 ],
+        [ "q_staff", -1 ],
+        [ "bat", -1 ],
+        [ "bat_metal", -1 ],
+        [ "bat_nerf", -1 ],
+        [ "baton-extended", -1 ],
+        [ "PR24-extended", -1 ],
+        [ "bowling_pin", -1 ],
+        [ "cane", -1 ],
+        [ "makeshift_cane", -1 ],
+        [ "club_wooden", -1 ],
+        [ "glass_macuahuitl", -1 ],
+        [ "club_wooden", -1 ],
+        [ "homewrecker", -1 ],
+        [ "makeshift_sap", -1 ],
+        [ "shillelagh_fake", -1 ],
+        [ "pipe", -1 ],
+        [ "stick", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "real_batons",
+    "type": "requirement",
+    "//": "Any one-handed weapon designed primarily to bash or smash things.  Reserved for quality items.",
+    "tools": [
+      [
+        [ "blackjack", -1 ],
+        [ "bwirebat", -1 ],
+        [ "shillelagh", -1 ],
+        [ "shillelagh_weighted", -1 ],
+        [ "shocktonfa_off", -1 ],
+        [ "tonfa", -1 ],
+        [ "tonfa_wood", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_flails",
+    "type": "requirement",
+    "//": "Any basic thing that can be used as a flail.",
+    "tools": [ [ [ "2h_flail_wood", -1 ] ] ]
+  },
+  {
+    "id": "real_flails",
+    "type": "requirement",
+    "//": "Any real heavy-ended bashing weapons that feature a long pole with an independently-swinging baton.",
+    "tools": [ [ [ "2h_flail_steel", -1 ] ] ]
+  },
+  {
+    "id": "basic_great_hammers",
+    "type": "requirement",
+    "//": "Any basic thing that works like a great hammer; unga bunga.",
+    "tools": [
+      [
+        [ "club_wooden_large", -1 ],
+        [ "hammer_sledge", -1 ],
+        [ "hammer_sledge_short", -1 ],
+        [ "hammer_sledge_heavy", -1 ],
+        [ "hammer_sledge_engineer", -1 ],
+        [ "lucern_hammerfake", -1 ],
+        [ "mace_pipe_large", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "real_great_hammers",
+    "type": "requirement",
+    "//": "Any real great hammers.  It's got to have a big blunt end on a long pole, directly attached to it.",
+    "tools": [
+      [
+        [ "warhammer", -1 ],
+        [ "lucern_hammer", -1 ],
+        [ "2h_club_kanabo_studded", -1 ],
+        [ "2h_club_kanabo_spiked", -1 ],
+        [ "lucern_hammer", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_maces",
+    "type": "requirement",
+    "//": "Any basic thing you can use as a mace.",
+    "tools": [
+      [
+        [ "mace_inferior", -1 ],
+        [ "mace_fake", -1 ],
+        [ "mace_trench", -1 ],
+        [ "mace_pipe", -1 ],
+        [ "mace_simple", -1 ],
+        [ "morningstar_fake", -1 ],
+        [ "morningstar_inferior", -1 ],
+        [ "nailbat", -1 ],
+        [ "nailboard", -1 ]
```

Comment:
I don't believe for a second that the nailboard is something appropriate for mace training... The nail bat... Well, the bats are currently considered "maces" for the sake of the martial art that uses maces, but I don't think they should be separated in the training recipes, they can either be used as maces for training, be used as poor batons/staffs or be used as both things, but in that case the practice recipes should include all the bats for when they are called, if you can use a nail bat for training, you definitely should be able to use a normal wood or metal bat too.